packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec

```diff
@@ -0,0 +1,40 @@
+%global repo_orgname project-receptor
+%global repo_name satellite-receptor-installer
```

Comment:
not strictly your playground (unless you are also upstream?) but I find this repo name and role name difference odd.

Reply:
meaning dashes vs underscores? not my choice, I have to believe that's best practices